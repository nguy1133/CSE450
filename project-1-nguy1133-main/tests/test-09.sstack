// One quote type can be used in a literal string with the other quote type.
PRINT "This is a single quote: '"
PRINT 'And this is a double quote: "'
