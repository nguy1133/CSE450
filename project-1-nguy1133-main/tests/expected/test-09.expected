This is a single quote: '
And this is a double quote: "
